features/netsocket/emac-drivers/TARGET_NUVOTON_EMAC/numaker_emac.cpp

```diff
@@ -222,15 +222,17 @@ bool NUMAKER_EMAC::link_out(emac_mem_buf_t *buf)
 {
     bool result;
```

Comment:
Not changed in this pull request, but noticed that this is left uninitialized on error cases. Should be set false here or on goto to error jumps.